src/Storage/Processor/Image.php

```diff
@@ -25,7 +25,7 @@ public function setWatermark(string $watermark): void
         $this->watermark = $watermark;
     }
 
-    public function generate(string $filename, string $cacheFilename = null)
+    public function generate(string $filename, string $cacheFilename = null): string
```

Comment:
it returns an array

Reply:
Please check again, really not sure with you